packages/devp2p/package.json

```diff
@@ -49,6 +49,7 @@
     "bl": "^1.1.2",
     "debug": "^4.3.3",
     "ethereumjs-util": "^7.1.5",
+    "ethereum-cryptography": "^1.0.3",
     "hi-base32": "^0.5.0",
```

Comment:
You should also remove this package, since ethereum-cryptography uses `@scure/base`, which contains audited implementation of base32/bech32.

Reply:
```
import { base32 } from '@scure/base';
```

Reply:
Would I need to add `@scure/base` as a new dependency in devp2p or can i access that from  ethereum-cryptography somehow?

Reply:
We've decided against exposing base32 because ETH apps don't need it. This is the first time I see an eth primitive requiring it. 

We can try exposing base32 in e-c, but i'd rather not to, at least for now. 

You can use the same scure/base version we're using in e-c for now, which is 1.1.0. You can use it as-is, or a range, I recommend ~1.1.0.